pkg/get/tools.go

```diff
@@ -519,6 +519,38 @@ https://github.com/inlets/inletsctl/releases/download/{{.Version}}/{{$fileName}}
 		https://github.com/digitalocean/doctl/releases/download/v{{.Version}}/doctl-{{.Version}}-{{$osStr}}-{{$archStr}}.{{$archiveStr}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "weaveworks",
+			Repo:        "eksctl",
+			Name:        "eksctl",
+			Version:     "v0.79.0",
+			Description: "Amazon EKS Kubernetes cluster management",
+			URLTemplate: `
+			{{$arch := ""}}
+			{{$extStr := "tar.gz"}}
+			{{- if eq .Arch "x86_64" -}}
+			{{$arch = "amd64"}}
+			{{- else if eq .Arch "aarch64" -}}
+            {{$arch = "arm64"}}
```

Comment:
Any reason for the different padding on this line?

Reply:
Sorry, It's an issue with the padding on my IDE. I'll fix this